Ecosystem Partners/Marketplace Guides/getting-started-with-waratek-locker.md

```diff
@@ -0,0 +1,153 @@
+{{{ 
+  "title": "Getting Started With Waratek Locker – Runtime Application Self Protection", 
+  "date": "10-21-2015", 
+  "author": "David Egan", 
+  "attachments": [], 
+  "contentIsHTML": false
+}}}
+
+### ![Waratek Logo](http://cdn.aws.waratek.com/wp-content/themes/waratek/images/logo.png)
+
+###Partner Profile
```

Comment:
Change all ###<HEADER TEXT> to ### <HEADER TEXT>
need a space between ### and text